ERCS/erc-6956.md

```diff
@@ -65,9 +65,7 @@ Lastly we want to mention two major side-benefits of using the proposed standard
 
 ### Related work
 
-We primarily aim to onboard physical or digital ASSETS into dApps, which do not signing-capabilities of their own (contrary to [ERC-5791](eip-5791.md) approach using crypto-chip based solutions). Note that we do not see any restrictions preventing to use ERC-5791 in combination with this standard, as the address of the crypto-chip qualifies as an ANCHOR.
-
-<!-- TO BE EXTENDED -->
+We primarily aim to onboard physical or digital ASSETS into dApps, which do not signing-capabilities of their own (contrary to the proposed `ERC-5791` using crypto-chip based solutions). Note that we do not see any restrictions preventing to use `ERC-5791` in combination with this standard, as the address of the crypto-chip qualifies as an ANCHOR.
```

Comment:
Either remove references to this ERC, or link to it. Please don't use backticks to hide it from the linter :P

```suggestion
We primarily aim to onboard physical or digital ASSETS into dApps, which do not signing-capabilities of their own (contrary to the proposed [ERC-5791](./eip-5791.md) using crypto-chip based solutions). Note that we do not see any restrictions preventing to use ERC-5791 in combination with this standard, as the address of the crypto-chip qualifies as an ANCHOR.
```

Reply:
I removed it, see https://github.com/ethereum/ERCs/pull/43/commits/300efa3da39ce0b274bbd1a65ce2f09e5adc9032#diff-a0d1f1c437d7ac8075ac9a94a3b6afdc880b6f17c8c017c73405e4b07f8618f9. Had to hide it because ERC-5791 is still in draft but they were [confident to move it to Review soon](https://ethereum-magicians.org/t/physical-backed-tokens/11350/20). Will re-add as soon as they move forward with their proposal